fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_FWC.js

```diff
@@ -380,4 +381,27 @@ class A32NX_FWC {
         }
         return true;
     }
+
+    updateRowRopWarnings() {
+        const w = Arinc429Word.fromSimVarValue('L:A32NX_ROW_ROP_WORD_1');
+
+        // ROW
+        SimVar.SetSimVarValue('L:A32NX_AUDIO_ROW_RWY_TOO_SHORT', 'bool', w.getBitValueOr(15, false));
+
+        // ROP
+        // MAX BRAKING, only for manual braking, if maximum pedal braking is not applied
+        const maxBrakingSet = SimVar.GetSimVarValue('L:A32NX_LEFT_BRAKE_PEDAL_INPUT', 'number') > 90 || SimVar.GetSimVarValue('L:A32NX_RIGHT_BRAKE_PEDAL_INPUT', 'number') > 90;
+        const maxBraking = w.getBitValueOr(13, false) && !maxBrakingSet;
+        SimVar.SetSimVarValue('L:A32NX_AUDIO_ROP_MAX_BRAKING', 'bool', maxBraking);
+
+        // SET MAX REVERSE, if not already max. reverse set and !MAX_BRAKING
+        const maxReverseSet = SimVar.GetSimVarValue('L:XMLVAR_Throttle1Position', 'number') < 0.1 && SimVar.GetSimVarValue('L:XMLVAR_Throttle2Position', 'number') < 0.1;
+        const maxReverse = (w.getBitValueOr(12, false) || w.getBitValueOr(13, false)) && !maxReverseSet;
+        SimVar.SetSimVarValue('L:A32NX_AUDIO_ROW_SET_MAX_REVERSE', 'bool', !maxBraking && maxReverse);
+
+        // At 80kt, KEEP MAX REVERSE once, if max. reversers deployed
+        const ias = SimVar.GetSimVarValue('AIRSPEED INDICATED', 'knots');
+        SimVar.SetSimVarValue('L:A32NX_AUDIO_ROP_KEEP_MAX_REVERSE', 'bool', (ias <= 80 && ias > 4) && (w.getBitValueOr(12, false) || w.getBitValueOr(13, false)));
+
```

Comment:
Hmm, we should not have any new stuff in here. We're trying to get rid of this file in favour of PseudoFWC (and eventual rust FWC).

Reply:
Meaning I should put this in PseudoFWC?

Reply:
moved to PseudoFWC